locales/en-US.json

```diff
@@ -210,6 +217,7 @@
   "save": "Save",
   "saved": "Saved",
   "search": "Search",
+  "seconds": "seconds",
```

Comment:
Shouldn't we keep it as "Seconds" and use a `toLowerCase` in the sentence?

Reply:
Agreed, no need to keep another instance of an identical string just for capitalization.